scripts/README.md

```diff
@@ -108,8 +108,10 @@ This is a helper script for Knative E2E test scripts. To use it:
    if the default values don't fit your needs:
 
    - `E2E_CLUSTER_REGION`: Cluster region, defaults to `us-central1`.
+   - `E2E_CLUSTER_BACKUP_REGION`: Backup cluster regions for retry, defaults to `us-west1 us-east1`. (Retry is only triggered when there is a stockout)
```

Comment:
```suggestion
   - `E2E_CLUSTER_BACKUP_REGIONS`: Space-separated list of regions to retry test cluster creation in case of stockout. Defaults to `us-west1 us-east1`.
```

Reply:
nit: For readability and consistency, use REGIONS in the var name.